test/oli/sections_test.exs

```diff
@@ -1255,4 +1255,28 @@ defmodule Oli.SectionsTest do
       assert Enum.at(Enum.at(hierarchy.children, 2).children, 1).numbering.index == 4
     end
   end
+
+  describe "list_user_enrolled_sections/1" do
+    setup do
+      student = insert(:user)
+      section_1 = insert(:section)
+      section_2 = insert(:section)
+
+      Sections.enroll(student.id, section_1.id, [ContextRoles.get_role(:context_learner)])
+      Sections.enroll(student.id, section_2.id, [ContextRoles.get_role(:context_learner)])
+      {:ok, %{student: student}}
+    end
+
+    test "returns sections for user", %{student: student} do
+      sections = Sections.list_user_enrolled_sections(student)
+      assert length(sections) == 2
```

Comment:
I think this assertion is not sufficient to test the expected behaviour of the function, since it implies the user is enrolled to 2 sections, but it does not guarantee the 2 sections match the sections the user was enrolled to.
If we expand the `setup` to return `section_1` and `section_2` we could extend the test to check the user is enrolled to those exact 2 sections, WDYT?